test/integration/asset_access_options_integration_test.rb

```diff
@@ -13,13 +13,13 @@ class AssetAccessOptionsIntegrationTest < ActionDispatch::IntegrationTest
 
     before do
```

Comment:
@tunylund Pretty much refactored all of this and nested everything under two big context with feature flag on/off